package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oat-sa/tao-core-sdk",
-  "version": "0.10.0",
+  "version": "0.11.0",
```

Comment:
shouldn't it be a breaking change since we've moved the JWT files we are breaking the compat ?

Reply:
Totally agree. I'm updating the version and the name of the branch.